ios/Samples/WesterosEmployees-hybrid/App/WesterosEmployees-hybrid/Resources/js/docs.js

```diff
@@ -0,0 +1,46 @@
+function modifyItem() {
+    
+    var urlIcons = {
+        pdf: "/documents/33300/39232/pdf.png/8c743026-cc37-07c3-1681-d7c9f4083b59",
```

Comment:
I think, it isn't possible because this file is injected to the webview and we don't have way to access swift code.